### PR TITLE
- iOS and Simulator build support

### PR DESCRIPTION
**Pull request by SSE4**
add support for iOS and iOS simulator cross-compilation
fixed issue with pipe which didn't fail if make command failed

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -172,6 +189,28 @@ def osx_build(self, config_options_string):
         else:
             command = "./config %s %s" % (config_options_string, m32_suff)
 
+        if self.settings.os == "iOS":
+            command = "./Configure iphoneos-cross %s" % config_options_string
```
Comment (lasote) [inline conanfile.py]: Create a new build_ios method and declare the 3 `call`, `find_sysroot` and `find_program`inside the ios method. You could also make a class if you want. I prefer to keep them isolated if they are only used for IOs.

Comment (SSE4) [reply conanfile.py]: sure

Comment (lasote) [reply conanfile.py]: I've uploaded a fix for the failing windows builds, could you update and push again with the little refactor I suggested?

Comment (SSE4) [reply conanfile.py]: sure!

**Pull request closed without merging**